src/installer/installer.go

```diff
@@ -26,14 +26,37 @@ import (
 )
 
 const (
-	InstallDir                  = "/opt/install-dir"
-	KubeconfigPathLoopBack      = "/opt/openshift/auth/kubeconfig-loopback"
-	KubeconfigPath              = "/opt/openshift/auth/kubeconfig"
-	minMasterNodes              = 2
-	dockerConfigFile            = "/root/.docker/config.json"
-	assistedControllerPrefix    = "assisted-installer-controller"
-	assistedControllerNamespace = "assisted-installer"
-	extractRetryCount           = 3
+	InstallDir                   = "/opt/install-dir"
+	KubeconfigPathLoopBack       = "/opt/openshift/auth/kubeconfig-loopback"
+	KubeconfigPath               = "/opt/openshift/auth/kubeconfig"
+	minMasterNodes               = 2
+	dockerConfigFile             = "/root/.docker/config.json"
+	assistedControllerPrefix     = "assisted-installer-controller"
+	assistedControllerNamespace  = "assisted-installer"
+	extractRetryCount            = 3
+	assistedInstallerSshManifest = "/opt/openshift/openshift/99_openshift-machineconfig_99-assisted-installer-master-ssh.yaml"
+	sshManifest                  = `
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  creationTimestamp: null
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-ssh
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    passwd:
+      users:
+      - name: core
+        sshAuthorizedKeys:
+        - {{.}}
+  fips: false
+  kernelArguments: null
+  kernelType:
+  osImageURL:
```

Comment:
I still don't understand why any of these keys are here. Do these have anything to do with setting an ssh key?

Reply:
Didn't reach to it yet :)

Reply:
done